src/web/collectors/CollectorRestartButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+import CollectorsActions from 'collectors/CollectorsActions';
+
+const CollectorsRestartButton = React.createClass({
+  propTypes: {
+    collector: React.PropTypes.object.isRequired,
+    backend: React.PropTypes.string.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      disabled: false,
+    };
+  },
+
+  componentDidMount() {
+    this.interval = setInterval(this._refreshButtonState, this.BUTTON_REFRESH);
```

Comment:
Good point, currently there are no more then two buttons used on the page but we should keep this in mind when we move it to another place.